float32 Ctrl_SW